.github/workflows/build.yml

```diff
@@ -15,24 +15,9 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - name: Cancel Outdated Builds
-        uses: styfle/cancel-workflow-action@0.11.0
-        with:
-          all_but_latest: true
-          access_token: ${{ github.token }}
-
       - name: Checkout Repository
         uses: actions/checkout@v3
 
-      - name: Install stable toolchain
```

Comment:
So, removing this means we're at the mercy of ... where does the toolchain get specified?

We're very inconsistent about our workflows, but at least a few use rust-toolchain.

https://github.com/dtolnay/rust-toolchain - this is probably a better choice than just leaving it to chance...

Reply:
https://github.com/dtolnay/rust-toolchain is a valid option. The toolchains are kept to stable by github, and I wasn't horribly worried about this project being out of date with stable for +/- 1 week.

Reply:
If we were building and exporting binaries I would agree, but I think it is mostly moot for a project like this.